packages/contracts-bedrock/src/periphery/faucet/authmodules/IFaucetAuthModule.sol

```diff
@@ -1,8 +1,8 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.0;
 
-// Contracts
-import { Faucet } from "src/periphery/faucet/Faucet.sol";
+// Libraries
+import { DripParameters } from "src/periphery/faucet/FaucetTypes.sol";
```

Comment:
Added this in order to fix a circular dependency between Faucet ->  IFaucetAuthModule -> Faucet